lib/editor/components/pattern/PatternStopCard.js

```diff
@@ -301,6 +369,94 @@ class PatternStopContents extends Component<Props, State> {
+            <FormGroup
+              controlId='pickupType'
+              bsSize='small'>
+              <ControlLabel
+                className='small'
+                title='Define the pickup method/availability at this stop.'
+              >
+                Pickup type
+              </ControlLabel>
+              <FormControl componentClass='select' placeholder='select' value={patternStop.pickupType} onChange={this._onPickupChange}>
+                {pickupDropoffOptions.map(o => (
+                  <option key={o.value} value={o.value}>{o.text}</option>
+                ))}
+              </FormControl>
+            </FormGroup>
+          </Col>
+          <Col xs={6}>
+            <FormGroup
+              controlId='dropoffType'
+              bsSize='small'>
+              <ControlLabel
+                className='small'
+                title='Define the dropff method/availability at this stop.'
+              >
+                Drop-off type
+              </ControlLabel>
+              <FormControl componentClass='select' placeholder='select' value={patternStop.dropOffType} onChange={this._onDropOffChange}>
+                {pickupDropoffOptions.map(o => (
+                  <option key={o.value} value={o.value}>{o.text}</option>
+                ))}
+              </FormControl>
+            </FormGroup>
+          </Col>
+        </Row>
+        {/* Continuous Stopping Pickup and drop off type selectors */}
+        <Row>
+          <Col xs={6}>
+            <FormGroup
+              controlId='continuous_pick-up'
+              bsSize='small'>
+              <ControlLabel
+                className='small'
+                title='Indicates whether a rider can board the transit vehicle anywhere along the vehicle’s travel path.'
+              >
+                  Continuous Pickup
+              </ControlLabel>
+              <FormControl
+                componentClass='select'
+                placeholder='select'
+                disabled={hasShapeId}
+                defaultValue={1}
+                value={patternStop.continuousPickup}
+                onChange={this._onContinuousPickupChange}
+              >
+                {pickupDropoffOptions.map(o => (
+                  <option key={o.value} value={o.value}>{o.text}</option>
+                ))}
+              </FormControl>
+            </FormGroup>
+          </Col>
+          <Col xs={6}>
+            <FormGroup
+              controlId='continuous_drop_off'
+              bsSize='small'>
+              <ControlLabel
+                className='small'
+                title='Indicates whether a rider can alight from the transit vehicle at any point along the vehicle’s travel path.'
+              >
+                  Continuous Drop off
+              </ControlLabel>
+              <FormControl
+                componentClass='select'
+                disabled={hasShapeId}
+                placeholder='select'
+                defaultValue={1}
```

Comment:
`defaultValue` is no longer needed (we set the `value` prop instead). (Also, sort props alphabetically)